features/step_definitions/i18n_steps.rb

```diff
@@ -1,3 +1,11 @@
+Given /^String "([^"]*)" corresponds to "([^"]*)"$/ do |translation, key|
+  *seq, last_key = key.split('.')
+  result = seq.reverse.inject({last_key.to_sym => translation}) do |temp_result, nested_key|
```

Comment:
Line is too long. [92/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.